integrations/salesforce-dmp/lib/index.js

```diff
@@ -117,6 +117,8 @@ SalesforceDMP.prototype.identifyV2 = SalesforceDMP.prototype.trackV2 = function
         }
         if (i === this.options.trackFireEvents.length - 1) return
       }
+    } else {
+      return
```

Comment:
This seems like a pretty big change? Do we have tests for this early return?

Reply:
Hmm, actually it seems like this early return is the meat of your fix. Maybe we could put a comment explaining why it needs to be there / what the consequences of leaving it out are?